File: Stylev/app/src/main/java/com/creator/stylev/util/ProgressUtil.java
package com.creator.stylev.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.creator.stylev.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2023/JejuGage/app/src/main/java/com/creator/jejugage/util/ProgressUtil.kt
package com.creator.jejugage.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.jejugage.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}